populate.py

```diff
@@ -56,37 +56,39 @@ def get_repository_info(owner, name):
     try:
         repository = client.repository(owner, name)
 
-        # store the repo info
-        info["name"] = name
-        info["owner"] = owner
-        info["language"] = repository.language
-        info["url"] = repository.html_url
-        info["stars"] = repository.stargazers_count
-        info["stars_display"] = numerize.numerize(repository.stargazers_count)
-        info["last_modified"] = repository.last_modified
-        info["id"] = str(repository.id)
-        info["objectID"] = str(repository.id)  # for indexing on algolia
-
-        # get the latest issues with the tag
-        issues = []
-        for issue in repository.issues(
-            labels=ISSUE_LABELS,
-            state=ISSUE_STATE,
-            number=ISSUE_LIMIT,
-            sort=ISSUE_SORT,
-            direction=ISSUE_SORT_DIRECTION,
-        ):
-            issues.append(
-                {
-                    "title": issue.title,
-                    "url": issue.html_url,
-                    "number": issue.number,
-                    "created_at": issue.created_at.isoformat()
-                }
-            )
-
-        info["issues"] = issues
-        return info
+        # check if repo has issues
+        if repository.has_issues:
```

Comment:
The counter on the page shows the number of issues tagged "good first issues", limited to the most recent 10. Just checking if the repository has issues or not wouldn't guarantee if the repository has at least one good first issue.

Instead of checking here, the number of issues can be inferred when adding (https://github.com/deepsourcelabs/good-first-issue/blob/master/populate.py#L107-L110), or the for loop iterating on `repository.issues` here can be moved up and the function can short-circuit if there are no issues with the filters are found.

Reply:
Got it! Thanks @sanketsaurav. Updated the MR now.